actions/push-to-gar-docker/action.yaml

```diff
@@ -44,6 +35,6 @@ runs:
         context: ${{ inputs.build_path }}
         push: ${{ github.event_name == 'push' }}
         tags: |
-          ${{ inputs.registry }}/${{ inputs.project }}/${{ inputs.repository }}/${{ inputs.image_name }}:${{ inputs.tag }}
+          ${{ inputs.tags }}
```

Comment:
trivial: Probably can be `tags: ${{ inputs.tags }}` (one line)

Reply:
true!